src/nvim/path.c

```diff
@@ -360,7 +360,7 @@ void add_pathsep(char *p)
   FUNC_ATTR_NONNULL_ALL
 {
   if (*p != NUL && !after_pathsep(p, p + strlen(p)))
-    strcat(p, PATHSEPSTR);
+    STRLCAT(p, PATHSEPSTR, MAXPATHL);
```

Comment:
By the way, what is the policy for STR macros? I see that this function has no reason to use `STRLCAT` in place of `xstrlcat` because `p` is `char*`, `PATHSEPSTR` is `char[2]` or `const char[2]` and `MAXPATHL` is an integer constant which thus is implicitly treated as `size_t` without needing any casts. I personally mostly omit upper-case macros. Also note that this function *did not* use macros, it used library `strcat`.